jdbc/maven_example/pom.xml

```diff
@@ -0,0 +1,18 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.actian.example</groupId>
+  <artifactId>Actian-JDBC-Test</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <name>Actian-JDBC-Test</name>
+  <url>https://github.com/ActianCorp/actian-code-samples</url>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.ingres.jdbc</groupId>
+      <artifactId>iijdbc</artifactId>
+      <version>12.0-4.4.4</version>
```

Comment:
dumb question, showing my ignorance. Is the version field required? Can this be omitted?

Reply:
I discovered that this works and changed it accordingly:

    <version>LATEST</version>

Thanks for the great tip!